src/screens/authentication/UsernameRequest/index.js

```diff
@@ -19,70 +42,66 @@ const SubmitIndicator = ({ submitting }) => (
     />
 )
 
-const Error = ({ error }) => (
-  error && (
-    <Body
-      secondary
-      style={styles.error}
-    >
-      {error}
-    </Body>
-  )
+const SubmitButton = ({ disabled, onPress, error }) => (
+  <PrimaryButton
+    onPress={onPress}
+    style={error ? { marginTop: verticalScale(20) } : { marginTop: verticalScale(40) }}
```

Comment:
Consider moving all styling into `styles.js` when it's a big component file like this. Something like `submitWithError` and `submitWithoutError` should suffice.